docs/source/blocks/introduction.md

```diff
@@ -14,14 +14,58 @@ The editor allows you to add, modify, reorder and delete blocks given your requi
 Blocks provide the user the ability to display content in an specific way, although they can also define behavior and have specific features.
 Blocks are composed of two basic (and required) components: the block edit and view components.
 
-By default, Volto ships with the most basic set of blocks: Title, Text, Image, Video, Maps, etc...
+(default-blocks)=
+
+## Default blocks
+
+Volto ships with a basic set of blocks, as described in the following list.
+For details of all their options, see {ref}`blocks/settings`.
+
+Title
+: Connected to the `title` content type's metadata field. It renders as the `h1` of the page. Cannot be removed by default, although this can be customized using `config.settings.requiredBlocks`.
+
+Text
+: The rich text block. This is the default type of block when you add a new block to the page.
+
+Image
+: Allows to insert an existing image or upload a new one. In case of uploading it gets added inside the content type (folderish) element. If the content is not folderish, it gets created side by side.
```

Comment:
```suggestion
: Allows insertion of an existing image or upload of a new one. In case of uploading, it gets added inside the content type (folderish) element. If the content is not folderish, it gets created side by side.
```